src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+    private static final String HEX_FORMAT = "%02x";
+
+    private HashUtil() {
+    }
+
+    public static String hashPassword(String password, byte[] salt) {
+        StringBuilder hashedPassword = new StringBuilder();
+        try {
+            MessageDigest messageDigest = MessageDigest.getInstance(CRYPTO_ALGORITHM);
+            messageDigest.update(salt);
+            byte[] digest = messageDigest.digest(password.getBytes());
+            for (byte b : digest) {
+                hashedPassword.append(String.format(HEX_FORMAT, b));
+            }
+        } catch (NoSuchAlgorithmException e) {
+            throw new IllegalStateException("Could`t create hash using SHA-512 algorithm", e);
```

Comment:
user your constant

Reply:
could you reuse your CRYPTO_ALOGIRTHM constant here?